org/collections/Employee.java

```diff
@@ -0,0 +1,226 @@
+        employeeList.add(new Employee("02", "Bob John", "IT", 60000, new Date(119, 8, 23), 35));
+        employeeList.add(new Employee("03", "Charlie Brown", "Finance", 55000, new Date(121, 1, 10), 28));
+        employeeList.add(new Employee("04", "Popov Ana", "Marketing", 65000, new Date(118, 11, 5), 40));
+        employeeList.add(new Employee("05", "Emma Frank", "IT", 70000, new Date(120, 7, 19), 32));
+        employeeList.add(new Employee("06", "Millie Bobby", "Sales", 45000, new Date(122, 3, 14), 29));
+        employeeList.add(new Employee("07", "Peaky Blinders", "HR", 52000, new Date(119, 6, 29), 34));
+        employeeList.add(new Employee("08", "Smith Christina", "Sales", 50000, new Date(122, 4, 15), 28));
+
+        //(ex.1)
+        int maxSalaryIT = getMaxSalaryForDepartment(employeeList, "IT");
+        System.out.println("Ex1. Max salary in IT department: " + maxSalaryIT);
+        System.out.println("-------------------");
+        //(ex.2)
+        double avgSalaryHR = getAverageSalaryForDepartment(employeeList, "HR");
+        System.out.println ("Ex2. Average salary in HR department: " + avgSalaryHR);
+        System.out.println("-------------------");
+        //(ex.3)
+        int numberOfEmployeeSales = getNumberOfEmployees(employeeList,"Sales");
+        System.out.println("Ex3. Number of Employees in Sales: " + numberOfEmployeeSales);
+        System.out.println("-------------------");
+        //(ex.4)
+        int numberOfEmployeeExceptIT = getNumberOfEmployeeExceptIT(employeeList, "IT");
+        System.out.println("Ex4. Number of Employees except IT department: " + numberOfEmployeeExceptIT);
+        System.out.println("-------------------");
+        //(ex.5)
+        System.out.println("Ex5. Group by Job Department: " + groupByJobDepartment(employeeList));
+        System.out.println("-------------------");
+        //(ex.6)
+        System.out.println("Ex6.");
+        printEmployeesJoinedAfter2020WithSalaryAbove1000(employeeList);
+        System.out.println("-------------------");
+        //(ex.7)
+        System.out.println("Ex7.");
+        Map<String, Integer> result = getLast3Employees(employeeList);
+        result.forEach((name, age) -> System.out.println(name + " : " + age));
+        System.out.println("-------------------");
+        //(ex.8)
+        int sumOfAllEmployees = calculateSumOfAllSalary(employeeList);
+        System.out.println("Ex8. Sum of all Salaries: " + sumOfAllEmployees);
+        System.out.println("-------------------");
+        //(ex.9)
+        Map<String, List<String>> salariesPerDepartment = returnAllSalaryPerDepartment(employeeList);
+        System.out.println("Ex9. Display all salaries per Department:");
+        salariesPerDepartment.forEach((department, salaries) ->
+        System.out.println("Department: " + department + "," + salaries));
+
+    }
+    //(ex.1)
+    public static int getMaxSalaryForDepartment(ArrayList<Employee> employees, String department) {
```

Comment:
The logic of getting the max is correct. 
But try to use streams. I will give you an example for this exercise and please refactor for the rest of the exercises:

all logic can be done like this: 

employees.stream().filter(f -> f.getDepartment().equals(department)).map(Employee::getSalary).max((Integer::compareTo)).get();

How it works:
1. You use your List variables
2. Call the stream() method
3. Call filter method to select only data you need that is equal to your department
4. Call Map method to return all Salary values
5. Call Max method to select the maximum value. Of course, you need a comparator logic to compare all values you have and will return the max one
6. Call get method to return int value